src/storage/src/hummock/sstable/mod.rs

```diff
@@ -114,6 +114,7 @@ impl Sstable {
             meta_offset: self.meta.meta_offset,
             stale_key_count: 0,
             total_key_count: self.meta.key_count as u64,
+            divide_version: 0,
```

Comment:
Where will we use this divide_version?

Reply:
Every time split the group. It is used to cancel compact task when one of the input sstable files was split into different group.